system/Common.php

```diff
@@ -474,6 +474,8 @@ function esc($data, string $context = 'html', ?string $encoding = null)
      * @param int $duration How long should the SSL header be set for? (in seconds)
      *                      Defaults to 1 year.
      *
+     * @return never|void
```

Comment:
@paulbalandan I probably wrote it somewhere, but isn't it wrong type?

> never is a return-only type indicating the function does not terminate. This means that it either calls [exit()](https://www.php.net/manual/en/function.exit.php), throws an exception, or is an infinite loop. Therefore, **it cannot be part of a [union type](https://www.php.net/manual/en/language.types.type-system.php#language.types.type-system.composite.union) declaration**. Available as of PHP 8.1.0.
https://www.php.net/manual/en/language.types.never.php

Reply:
Honestly, I didn't know `never` cannot be used in unions. TIL. I'll change.

Reply:
No problem. I also wrote `@return false|never|string`.